modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/extractors.kt

```diff
@@ -51,6 +51,17 @@ object AssetDefinitionsExtractor : ResultExtractor<List<AssetDefinition>> {
     }
 }
 
+object U32Extractor : ResultExtractor<UInt> {
+    override fun extract(result: QueryResult): UInt {
+        return when (val value = result.value) {
+            is Value.U32 -> value.cast<Value.U32>().u32
```

Comment:
This `value.cast<Value.U32>().u32` is unnecessary. A s soon as `is Value.U32` arm is matched, the value will be implicitly casted to `Value.U32`